source/02_getting_started/05_load_data/index.rst

```diff
@@ -89,5 +93,60 @@ Finally, the composition can be sent to the openEHR server:
         CompositionEndpoint compositionEndpoint = openEhrClient.compositionEndpoint(ehr);
         UUID compositionId = compositionEndpoint.saveCompositionEntity(highmedCardioMonitoringV1);
 
+Flat Format
+^^^^^^^^^^^
+Another alternative to using the Client Library is to use a `Simplified Data Template <https://specifications.openehr.org/releases/ITS-REST/latest/simplified_data_template.html>`_ also known as the "Flat format". 
+In particular, we'll be looking at the simplified IM Simplified Data template (simSDT) which is based on the web template format created by Marand for the Better platform. 
+The first thing you need is to get the Web Template version of the Template. The ADL Designer tool allows you to export templates as Web Templates.
+An example of a simple Body Temperature Web Template (borrowed from `EhrScape Examples <https://www.ehrscape.com/examples.html>`_) would look like this:
+
+ .. image:: images/webTemplate.png
+   :alt: alternate text
+   :align: center
+
+Next, we create the composition from the Web Template as a simple key-value pair with the keys being a path 
+obtained by concatenating the :code:`id` of each level delimited by a :code:`/`. The last segment is the suffix and uses :code:`|` as a delimiter. 
+
+For example, in the above image all the :code:`id` to be concatenated are highlighted in red.
+
+So the paths built from the above example would look like:
+
+:code:`vital_signs/body_temperature/any_event/temperature|magnitude`
+:code:`vital_signs/body_temperature/any_event/temperature|unit`
+
+The value of these above keys would the actual data. Representing this in JSON would look like:
+
+.. code-block:: JSON
+
+                {
+                "vital_signs/body_temperature/any_event/temperature|magnitude": 92,
+                "vital_signs/body_temperature/any_event/temperature|unit": "°C"
+                }
+
+However, since the cardinality of the :code:`body_temperature` and :code:`any_event` elements are :code:`-1` it means that
+the composition can have an infinite number of :code:`body_temperature` and :code:`body_temperature` recorded in the same composition.
+To resolve this, we have to index the path like so:
+:code:`vital_signs/body_temperature:0/any_event:0/temperature|magnitude`
+:code:`vital_signs/body_temperature:0/any_event:0/temperature|unit`
+
+With these paths, and more context data, a composition with multiple recordings of body temperature will look like
```

Comment:
Can you add `:` at the end of the line for consistency?

Reply:
Done